eslint.config.mjs

```diff
@@ -149,6 +149,7 @@ export default tseslint.config(
                 {
                     allowNumber: true,
                     allowBoolean: true,
+                    allowArray: true,
```

Comment:
not sure arrays were not already allowed but this is needed so the endpoint can return `{ deletedKeys: string[] }`

Reply:
ok. in this case let's keep the rule as it is and have an disable exception in the endpoint since we know there it is a array of string